wgpu/examples/framework.rs

```diff
@@ -267,14 +267,7 @@ fn start<E: Example>(
     }: Setup,
 ) {
     let spawner = Spawner::new();
-    let mut config = wgpu::SurfaceConfiguration {
-        usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
-        format: surface.get_supported_formats(&adapter)[0],
-        width: size.width,
-        height: size.height,
-        present_mode: wgpu::PresentMode::Fifo,
-        alpha_mode: surface.get_supported_alpha_modes(&adapter)[0],
-    };
+    let mut config = wgpu::SurfaceConfiguration::from_size(size.width, size.height);
```

Comment:
This won't actually work as Srgb isn't available on the web.

Reply:
Added a wasm32 target compile condition.

Reply:
I think we need to add the adapter so we can call get_surface_preferred_formats - these kinds of blanket defaults make me uneasy with the general complexity of WSI